build_js.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+git submodule update --init --recursive
+
+DATA_DIR="$(pwd)/data/"
+
+mkdir -p build_js
+cd build_js
+
+cmake ../src \
+    -DBUILD_WEBGL=ON \
+    -DBUILD_GUI_VIEWERS=ON \
+    -DBUILD_PYTHON_BINDINGS=OFF \
+    -DBUILD_DATATOOL=OFF \
+    -DBUILD_PTEX_SUPPORT=OFF \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_PREFIX_PATH="$EMSCRIPTEN" \
+    -DTARGET_GLES2=OFF \
+    -DTARGET_WEBGL=ON \
```

Comment:
No such flag -- Magnum detects Emscripten based on the toolchain file and then enables WebGL implicitly.